content/en/APIUseCases/get-findings.md

```diff
@@ -0,0 +1,259 @@
+        "suggested_fix": "Ensure this...",
+        "prerequisites": "Credentials are needed",
+        "pentest_id": "pt_PEtv4dqnwGV2efZhLw3BM5",
+        "http_request": "HTTP GET / ...",
+        "asset_id": "as_HcChCMueiPQQgvckmZtRSd",
+        "log": [
+          {
+            "action": "created",
+            "timestamp": "2021-04-01T15:13:24.322Z"
+          },
+          {
+            "action": "likelihood_changed",
+            "value": 4,
+            "timestamp": "2021-04-01T15:14:05.856Z"
+          },
+          {
+            "action": "impact_changed",
+            "value": 5,
+            "timestamp": "2021-04-01T15:14:05.856Z"
+          },
+          {
+            "action": "state_changed",
+            "value": "need_fix",
+            "timestamp": "2021-04-01T15:14:06.757Z"
+          },
+          {
+            "action": "state_changed",
+            "value": "check_fix",
+            "timestamp": "2021-04-01T15:14:57.845Z"
+          }
+        ],
+        "state": "check_fix",
+        "created_at": "2022-09-26T18:35:18.759Z",
+        "updated_at": "2022-09-26T18:36:57.462Z",
+        "attachments": [
+          {
+            "id": "at_LA5GcEL4HRitFGCHREqmzL",
+            "file_name": "rainbow.jpeg",
+            "download_url": "https://s3.amazonaws.com/acmecorp/uploads/attachment/file/12345/rainbow.jpeg?something=1"
+          }
+        ]
+      },
+      "links": {
+        "ui": {
+          "url": "https://api.cobalt.io/links/eyJ0eXBlIjoic29tZXRoaW5nIiwib3JnU2x1ZyI6ImNvYmFsdCIsInBlbnRlc3RUYWciOiJz="
+        }
+      }
+    }
+  ],
+  "pagination": {
+    "next_page": "/findings?cursor=a1b2c3d4",
+    "prev_page": "/findings?cursor=4d3c2b1a"
+  }
+}
+```
+{{% /expand %}}
+</br>
+
+For more information on each parameter, see our API reference documentation on
+how to [get all findings](https://docs.cobalt.io/v2/#get-all-findings).
+
+If the command is successful, you'll see
+
+| Message    | Meaning          |
+|------------|------------------|
+| 200 OK | Findings for all pentests retrieved    |
+
+For a list of error codes, see the [Errors](https://docs.cobalt.io/v2/#errors)
+section of our API reference.
+
+## Step 4: Import Findings to Google Sheets
+
+Now that you have findings data in JSON format, you can import it to the spreadsheet software of your choice.
+
+As an example, let's import findings to Google Sheets.
+
+1. In Google Sheets, select **Extensions** > **Apps Script**.
+1. Use the [Class UrlFetchApp](https://developers.google.com/apps-script/reference/url-fetch/url-fetch-app) service to import raw findings data. Refer to the [Class SpreadsheetApp](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app) reference for details.<br>
+1. Run the following script to download and import Cobalt findings. Replace `{{% api-token %}}` and `{{% org-token %}}` with your values.<br>
+
+    ```js
+    function FETCH_DATA() {
+      var url = 'https://api.cobalt.io/findings';
```

Comment:
@nktrk I removed `pentest` from the script to fetch findings for all pentests. The script works.